src/DeviceProvisioningServices/DeviceProvisioningServices/Models/PSDeviceRegistrationState.cs

```diff
@@ -76,4 +76,7 @@ public class PSDeviceRegistrationState
         [JsonProperty(PropertyName = "etag", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public string ETag { get; internal set; }
     }
+
+    public class PSDeviceRegistrationStates : PSDeviceRegistrationState
+    { }
```

Comment:
Why is this PSDeviceRegistrationStates neede?

Reply:
The command will return either a single object or multiple objects, So having different output types is required to mention in .ps1xml format file to display the output of multiple objects in table format and single object in list format.
`PSDeviceRegistrationStates: Use to display the output in table format`
`PSDeviceRegistrationState: Use to display the output in list format`